<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>userEdit</class>
 <widget class="QDialog" name="userEdit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>479</width>
    <height>304</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Uredi korisnika</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../ipi.jpg</normaloff>../ipi.jpg</iconset>
  </property>
  <widget class="QTableView" name="tbl_userEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>441</width>
     <height>192</height>
    </rect>
   </property>
   <property name="sizeAdjustPolicy">
    <enum>QAbstractScrollArea::AdjustToContentsOnFirstShow</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_edit_user">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>260</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Uredi korisnika</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_edit_predmet">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>260</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Uredi predmete</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="chck_siguran">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>270</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_delete">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>260</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Obriši učenika</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_userAdd">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Dodaj korisnika</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_nazad">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>10</y>
     <width>80</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Nazad</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_refresh">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>10</y>
     <width>80</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Refresh</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>chck_siguran</sender>
   <signal>toggled(bool)</signal>
   <receiver>btn_delete</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>374</x>
     <y>315</y>
    </hint>
    <hint type="destinationlabel">
     <x>342</x>
     <y>316</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
